arangod/RocksDBEngine/RocksDBBuilderIndex.h

```diff
@@ -23,15 +23,88 @@
 
 #pragma once
 
+#include "Basics/ConditionLocker.h"
```

Comment:
I don't think the `ConditionLocker` is required anymore, as you have switched to `std::condition_variable`.
```suggestion
```